backend/src/main/java/com/bakdata/conquery/models/query/concept/specific/CQReusedQuery.java

```diff
@@ -65,9 +72,10 @@ public QPNode createQueryPlan(QueryPlanContext context, ConceptQueryPlan plan) {
 
 	@Override
 	public void resolve(QueryResolveContext context) {
+		query = ((ManagedQuery) context.getDatasetRegistry().getMetaRegistry().resolve(queryId));
 		resolvedQuery = query.getQuery();
 
-		// Yey recursion, because the query might consists of another CQReusedQuery or CQExternal
+		// Yey recursion, because the query might consist of another CQReusedQuery or CQExternal
```

Comment:
You sure?

Reply:
100%

Reply:
aber was ist mit he she it das s muss mit?